conf/messages.en

```diff
@@ -1088,3 +1088,11 @@ transport.transportMeans.departure.inlandMode.waterway = Inland waterway
 transport.transportMeans.departure.inlandMode.unknown = Unknown (own propulsion)
 transport.transportMeans.departure.inlandMode.checkYourAnswersLabel = Which inland mode of transport are you using?
 transport.transportMeans.departure.inlandMode.error.required = Select which inland mode of transport you’re using
+
+transport.transportMeans.departure.vehicleCountry.title = What country is this vehicle registered to?
+transport.transportMeans.departure.vehicleCountry.heading = What country is this vehicle registered to?
+transport.transportMeans.departure.vehicleCountry.hint = Enter the country or code, like AT or Austria.
+transport.transportMeans.departure.vehicleCountry.placeholder = Select a nationality
+transport.transportMeans.departure.vehicleCountry.change.hidden = What country is this vehicle registered to?
```

Comment:
Remember that these messages are always preceded by "Change " so we'd probably want this to be something like "the country the vehicle is registered to". Obviously this will get ironed out when we build the CYA page but it should do for now.